### PR TITLE
fix: errors with missing posix extension

### PR DESCRIPTION
**Pull request by murrant**
It should be an optional dependency

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: LibreNMS/Validations/Php.php [hunk 1]
```diff
@@ -63,6 +63,13 @@ private function checkExtensions(Validator $validator)
                 $validator->fail("Missing PHP extension: $extension", "Please install $extension");
             }
         }
+
+        $suggested_extensions = array('posix' => 'php-process');
+        foreach ($suggested_extensions as $extension => $packages) {
+            if (!extension_loaded($extension)) {
+                $validator->warn("Missing optional PHP extension: $extension", "It is suggested you install $packages or the one that matches your php version");
```
Comment (laf) [inline LibreNMS/Validations/Php.php]: Will this not just lead to people asking about install this and should we not just have it as part of the install docs?

Comment (murrant) [reply LibreNMS/Validations/Php.php]: Yeah, probably would be worth adding it to the install docs.  No change is necessary on Ubuntu.

Comment (laf) [reply LibreNMS/Validations/Php.php]: As in it's already installed?

Comment (murrant) [reply LibreNMS/Validations/Php.php]: As in for Debian/Ubuntu, it is part of php core.

**Pull request merged**